lib/feature-collection.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
```

Comment:
@amishas157 can we rename this function to lib/getVectorTile or something to indicate that this is fetching a tile and returning features?